packages/lesswrong/client/serverSentEventsClient.ts

```diff
@@ -22,11 +25,14 @@ function recreateEventSource() {
   console.log("Connecting to server-sent events");
   serverSentEventSource = new EventSource("/api/notificationEvents");
   serverSentEventSource.onerror = (errorEvent) => {
```

Comment:
FWIW @jimrandomh I'm not convinced this will ever fire more than once from reading the spec, and was going to ask about that, but of course you may have live experience and it's harmless to add an extra check.